python/lsst/ts/standardscripts/maintel/laser_tracker/align.py

```diff
@@ -35,8 +35,8 @@
 
 # TODO (DM-38880) - Subsitute by class in idl.enum when available
```

Comment:
This ticket is merged, can you replace these with the AlignComponent class there?

Reply:
No, not yet.. only when the new cycle is released..